includes/Story_Renderer/Image.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Class Image
+ *
+ * @package   Google\Web_Stories\Story_Renderer
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Google\Web_Stories\Story_Renderer;
+
+use Google\Web_Stories\Model\Story;
+
+/**
+ * Class Image
+ *
+ * @package Google\Web_Stories\Story_Renderer
+ */
+class Image {
```

Comment:
`Image` is misleading, because if there is no poster image it will be just a link.

Reply:
Would would you recommend?

Reply:
I think I used `Static` before but you mentioned that has downsides as well.. So... 🤷

Reply:
let's keep it for now; just wanted to raise this.

Reply:
I suggest simple html or simple preview. I don't like either of them, but there we are.

Reply:
Let's leave it as-is for now